test/srv_seedlist/replica-set/srv-service-name.json

```diff
@@ -0,0 +1,16 @@
+{
+  "uri": "mongodb+srv://test22.test.build.10gen.cc/?srvServiceName=customname",
+  "seeds": [
+    "localhost.test.build.10gen.cc:27017",
+    "localhost.test.build.10gen.cc:27018"
+  ],
+  "hosts": [
+    "localhost:27017",
+    "localhost:27018",
+    "localhost:27019"
+  ],
+  "options": {
+    "ssl": true,
+    "srvServiceName": "customname"
```

Comment:
Does the srv_seedlist test suite actually check the `srvServiceName` option here? EG would the test fail with 
`"srvServiceName": "wrongname"`